.github/CODEOWNERS

```diff
@@ -277,6 +277,13 @@ packages/core/saved-objects/core-saved-objects-server @elastic/kibana-core
 packages/core/saved-objects/core-saved-objects-server-internal @elastic/kibana-core
 packages/core/saved-objects/core-saved-objects-server-mocks @elastic/kibana-core
 packages/core/saved-objects/core-saved-objects-utils-server @elastic/kibana-core
+packages/core/security/core-security-browser @elastic/kibana-core
```

Comment:
I'd say we'll only need to do that once the implementation itself is extracted, for the packages/places it was extracted to.

Reply:
It might make sense to make as co-owners for `packages/core/security/core-security-common` at least since we completely moved certaon types from the security package in this case.